src/DynamoCore/Configuration/PathManager.cs

```diff
@@ -491,13 +493,14 @@ private string TransformPath(string root, string extension)
         private static string GetSamplesFolder(string dataRootDirectory)
         {
             var versionedDirectory = dataRootDirectory;
-            if (!Directory.Exists(versionedDirectory))
+            if (!Directory.Exists(versionedDirectory)
+                || !Directory.Exists(Path.Combine(versionedDirectory, SamplesDirectoryName)))
             {
                 dataRootDirectory = Directory.GetParent(versionedDirectory).FullName;
             }
```

Comment:
This might explain it a little better to make the code more maintainable:

```csharp
if (!Directory.Exists(versionedDirectory)
{
    // Try to see if folder "%ProgramData%\Dynamo\%ProdName%\1.1" exists, if it 
    // does not, then root directory would be "%ProgramData%\Dynamo\%ProdName%".
    // 
    dataRootDirectory = Directory.GetParent(versionedDirectory).FullName;
}
else if (!Directory.Exists(Path.Combine(versionedDirectory, SamplesDirectoryName)))
{
    // If the folder "%ProgramData%\Dynamo\%ProdName%\1.1" exists, then try to see 
    // if the folder "%ProgramData%\Dynamo\%ProdName%\1.1\samples" exists. If it 
    // doesn't exist, then root directory would be "%ProgramData%\Dynamo\%ProdName%".
    // 
    dataRootDirectory = Directory.GetParent(versionedDirectory).FullName;
}
```